pkg/internal/testing/process/process_test.go

```diff
@@ -352,16 +352,7 @@ var _ = Describe("Init", func() {
 })
 
 var simpleBashScript = []string{
-	"-c",
-	`
-		i=0
-		while true
-		do
-			echo "loop $i" >&2
-			let 'i += 1'
-			sleep 0.2
-		done
-	`,
+	"-c", "tail -f /dev/null",
```

Comment:
@alvaroaleman @inteon I noticed the bash shells were never getting cleaned up, it seems this hot loop was the cause, because it never catches that it should go away.

Reply:
...what is this used for?

Reply:
Block forever basically

Reply:
It's just for testing